boards/BMS/Inc/Io/Io_SoftwareWatchdog.h

```diff
@@ -1,8 +1,9 @@
 #pragma once
 
-#include "App_CanTx.h"
 #include "Io_SharedSoftwareWatchdog.h"
 
+struct BmsCanTxInterface;
```

Comment:
Replace `App_CanTx.h` with `struct BmsCanTxInterface;` which should improve compilation time and make sure this file doesn't pull it a bunch of unnecessary code from `App_CanTx.h`